plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/DeployImagePage.java

```diff
@@ -225,7 +227,14 @@ public IStatus validate(Object value) {
 			.observe(model))
 			.in(dbc);
 		ControlDecorationSupport.create(
-			selectedConnectionBinding, SWT.LEFT | SWT.TOP, null, new RequiredControlDecorationUpdater(true));	}
+			selectedConnectionBinding, SWT.LEFT | SWT.TOP, null, new RequiredControlDecorationUpdater(true));
+		
+        StyledText newDockerConnectionLink = StyledTextUtils.emulateLinkWidget("<a>New Docker connection</a>", new StyledText(parent, SWT.WRAP));
+        GridDataFactory.fillDefaults()
+            .align(SWT.LEFT, SWT.CENTER).indent(8, 0)
+            .applyTo(newDockerConnectionLink);
+        StyledTextUtils.emulateLinkAction(newDockerConnectionLink, r->onNewDockerConnectionClicked());
+		}
 	
 	private void createConnectionControl(Composite parent, DataBindingContext dbc) {
```

Comment:
The connection control is no longer present. Can you please rebase against master?